cockroachdb/templates/cronjob-ca-certSelfSigner.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if and .Values.tls.certs.selfSigner.enabled .Values.tls.certs.selfSigner.rotateCerts }}
+  {{- if eq false .Values.tls.certs.selfSigner.caProvided }}
```

Comment:
Can we put this guard on the outside? If the user doesn't provide a CA we shouldn't be producing this template. Also does helm support `not` [seems like it does](https://helm.sh/docs/chart_template_guide/function_list/#not) could we rewrite this as 
`{{- if not .Values.tls.certs.selfSigner.caProvided }}` so it's a bit easier to parse?

Reply:
Done